[OSDOCS-7716]: Follow Up Feedback for https://issues.redhat.com/browse/OSDOCS-3562 (#65849)
openshift/openshift-docs: modules/sre-cluster-access.adoc

The information presented below is an overview of the process an SRE must perform to access a customer's cluster.

** SRE requests a refreshed ID token from the Red Hat SSO (Cloud Services). This request is authenticated. The token is valid for fifteen minutes. Once the token expires, the user can refresh the token again and receive a new token. The ability to refresh to a new token is indefinite; however, the ability to refresh to a new token is revoked after thirty days of inactivity.

// ** SRE sends a request tunneled through the Red Hat VPN. This request is made via Corporate Identity and Access Management system (RH IAM); authentication is multi-factor (made up of a password and an ephemeral one-time token). Once the SRE authenticates and is allowed access to the orchestration and management systems, the authorization is managed by Red Hat corporate directory services. The use of RH IAM enables SREs to be managed internally per organization via groups and existing on-boarding/off-boarding processes. Changes to the orchestration and management systems require many layers of approval and are maintained by strict company policy.

> ** SRE connects to the Red Hat VPN. The authentication to the VPN is performed by the Red Hat Corporate Identity and Access Management system (RH IAM). The use of RH IAM enables SREs to be managed internally per organization via groups and existing on-boarding/off-boarding processes, and is multi-factor. Once authenticated and connected, the SRE can access the cloud services fleet management plane. Changes to the cloud services fleet management plane require many layers of approval and are maintained by strict company policy.

Reviewer: ```suggestion
** SRE connects to the Red Hat VPN. The authentication to the VPN is completed by the Red Hat Corporate Identity and Access Management system (RH IAM). With RH IAM, SREs can be managed internally per organization via groups and existing on-boarding/off-boarding processes, and is multi-factor. After an SRE is authenticated and connected, the SRE can access the cloud services fleet management plane. Changes to the cloud services fleet management plane require many layers of approval and are maintained by strict company policy.
```
I realize that you didn't write this paragraph as part of this PR, but if possible, consider the edits here. A few things to note:
- IBM Style advises the use of "completed" instead of "performed". (Search for "perform" on the [Word Usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage) page for detailed guidance.
- I reworded the second sentence to avoid the use of "enables," which is anthropomorphic language. See [Anthropomorphism](https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism) in IBM Style.
- Also in the second sentence, I'm not sure what "and is multi-factor" refers to. Would it be accurate to reword the sentence as follows? "RH IAM is a multi-factor system that manages SREs interally per organization via groups and existing on-boarding/off-boarding processes."
- Replaced "Once" with "After" per IBM Style.

> ** Once authorized, SRE logs into the fleet management plane and receives a service account token that the fleet management plane created. The token is valid for fifteen minutes. Once the token is no longer valid, it is deleted.

Reviewer: ```suggestion
** After authorization is complete, the SRE logs into the fleet management plane and receives a service account token that the fleet management plane created. The token is valid for 15 minutes. After the token is no longer valid, it is deleted.
```

** With access granted to the fleet management plane, SRE uses various methods to access clusters, depending on network configuration.

> *** Accessing a private or public cluster: Request is sent through a specific Network Load Balancer (NLB) using an encrypted HTTP connection on port 6443.

Reviewer: ```suggestion
*** Accessing a private or public cluster: Request is sent through a specific Network Load Balancer (NLB) by using an encrypted HTTP connection on port 6443.
```
Guidance from IBM Style Word Usage about the term _using_: "To avoid ambiguity, replace this gerund with either _by using_ or _that uses_."
